charts/karavi-observability/crds/cert-manager.crds.yaml

```diff
@@ -0,0 +1,4414 @@
+# Copyright 2022 The cert-manager Authors.
```

Comment:
Should this cert-manager.crds.yaml be in a common location? If user doesn't select observability module but selects app-mobility, will cert-manager get installed successfully?

Reply:
Cert manager is added as dependency in Umbrella chart so it will get installed separately and crds are added in crds folder so they will get installed along with the chart and will get skipped if already present